Dockerfile

```diff
@@ -1,12 +1,16 @@
-FROM openjdk:8u242-jre
-
-run mkdir /app
+FROM ethereum/solc:0.4.24 as builder
```

Comment:
for dockerfiles, please add a line-by-line commenting about what each line is doing.